client/public/static/js/account/my_recipes.js

```diff
@@ -252,6 +252,15 @@ $(document).ready(function () {
                           <a href="/recipes/recipe_details?source=${recipeSource}&sourceUrl=${recipeSourceUrl}&uri=${recipeUri}">
                               <img src="${recipeImage}" alt="${recipeName}" title="View more about ${recipeName}">
                           </a>
+                            <div class="recipe-dropdown">
```

Comment:
Is this functionality specifically only to a single recipe? I could see us doing this exactly functionality in the recipes view as well but no action yet

Reply:
Yea, the dropdowns are on each and every separate recipe.